bin/watch-packages-and-rebuild.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { spawnSync } = require( 'child_process' );
+const chokidar = require( 'chokidar' );
+
+const packagesDirectoryPath = path.join( '.', 'packages' );
+const watcher = chokidar.watch( packagesDirectoryPath, {
+	ignored: /^\./,
+	persistent: true,
+	ignoreInitial: true,
+} );
+watcher.on( 'change', handleChange ).on( 'add', handleChange ).on( 'unlink', handleChange );
+
+let rebuildQueue = [];
+
+setTimeout( processRebuildQueue, 100 );
```

Comment:
I'm a bit concerned about this recurring execution. Looks like it will waste many CPU cycles doing nothing.

An alternative approach would be to use debounce:

```
const debounce = require('lodash/debounce');
const debouncedProcessRebuildQueue = debounce(processRebuildQueue,100);

function handleChange() {
     //...
    rebuildQueue.push( packageName );
    // It will invoke `processRebuildQueue` 100ms after the last execution of `handleChange`
	debouncedProcessRebuildQueue();
}
```

Reply:
I agree. If you remember my packages caching PR, packages folder is only modified in 8.4% of the commits. It'd be a big loss if we loop here. Debouncing sounds good.